src/include/daos_srv/bio.h

```diff
@@ -354,6 +354,7 @@ struct bio_dev_info {
 	int		       *bdi_tgts;
 	char		       *bdi_traddr;
 	uint32_t		bdi_dev_type;	/* reserved */
+	uint32_t		bdi_dev_roles;	/* reserved */
```

Comment:
Is it really reserved? seems like it's used in srv_query.c

Reply:
I think it intent to replace bdi_dev_type with bdi_dev_roles here..

Reply:
I will update on the next re-push, or as part of a subsequent PR.